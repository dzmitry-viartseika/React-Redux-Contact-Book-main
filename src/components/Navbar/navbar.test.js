import React from 'react';
import Adapter from 'enzyme-adapter-react-16';
import { shallow, configure } from 'enzyme';
import toJson from "enzyme-to-json";
import Navbar from './index';

configure({adapter: new Adapter()});

test('renders correctly Navbar component', () => {
    const component = shallow (<Navbar />);
    const wrapper = component.find('.navbar');
    expect(toJson(wrapper)).toMatchSnapshot();
});