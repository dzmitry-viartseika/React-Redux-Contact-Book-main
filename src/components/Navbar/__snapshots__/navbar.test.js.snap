// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly Navbar component 1`] = `
Array [
  <div
    className="navbar col-md-12 bg-dark py-2"
  >
    <nav
      className="navbar bg-dark navbar-dark"
    >
      <Link
        className="navbar-brand ml-5"
        to="/"
      >
        React Redux Contact Book
      </Link>
    </nav>
  </div>,
  <nav
    className="navbar bg-dark navbar-dark"
  >
    <Link
      className="navbar-brand ml-5"
      to="/"
    >
      React Redux Contact Book
    </Link>
  </nav>,
]
`;
